src/components/admin/UserTable.vue

```diff
@@ -1,20 +1,21 @@
 <script setup lang="ts">
   import PasswordReset from '@/components/admin/PasswordReset.vue'
-  import { useI18n } from 'vue-i18n'
+  import { Composer, useI18n } from 'vue-i18n'
+  import { Person } from '@/stores/PersonStore'
 
   /* this block is necessary to introduce a table header type to shut up typescript when defining table headers */
```

Comment:
```suggestion
  /* this block is necessary to introduce a table header type for defining table headers
      watch source for updates: https://stackoverflow.com/a/75993081/4790594
   */
```
this was implemented as a workaround. if we want to watch the source for updates on newer vuetify versions, I added the link